### ISSUE TITLE
Wrong line endings in php files (Development Standard)

### ISSUE THREAD

**Issue by euantorano** [mybb/mybb#2157]
I noticed that some of the files of the project don't follow the http://docs.mybb.com/1.8/development/standards/

MyBB Development Standards Wrote:New Lines

All new lines should be denoted by the UNIX line feed (\n, also known as "LF") and not include the Windows carriage return (\r, also known as "CR"). CR and CRLF are not permitted; LF is the only permitted line ending.

The full list of the 1.8.5 release with wrong line endings (crlf instead of lf only):

admin\styles\default\style.php
inc\3rdparty\2fa\GoogleAuthenticator.php
inc\db_sqlite.php
install\resources\upgrade25.php
member.php
usercp2.php

It would be nice to see these, maybe with 1.8.6, corrected to follow the UNIX standard of lf only like the rest of the files. I know that such a change isn't easy to do (especially for the last two files), because it may result in difficulties for merges from other developers/forks in some cases.

Original thread: [Wrong line endings in php files (Development Standard)](http://community.mybb.com/thread-173279.html)

**Comment by JN-Jones** [comment]
That change should be in 1.8.6 as the changed files packages will likely include most files anyways due the doc block changes. Also note that other standards (mainly tabs vs spaces) are also ignored in some places. And note that the GoogleAuthenticator file should be updated first.

**Comment by euantorano** [comment]
Agreed. We should add a `.gitattributes` file to force line-endings too.

**Comment by JN-Jones** [comment]
Do we still want to fix this @mybb/management @mybb/developers? As mentioned this may result in quite a big changed files package which we may not want at this point of development.

**Comment by euantorano** [comment]
It's not causing a problem exactly as is, is it? If it was causing errors or anything I'd say we should fix it. If it's just a minor annoyance, it's likely not worth the hassle.

**Comment by JN-Jones** [comment]
No issues afaik, just us not following our own standards.
